README.md

```diff
@@ -52,14 +59,17 @@ Once installed, the Fluent Bit Operator provides the following features:
 
 ## Overview
 
-Fluent Bit Operator defines five custom resources using CustomResourceDefinition (CRD):
+Fluent Operator is mainly composed of Fluent Bit Operator and Fluentd Operator.
+
+### Fluent Bit Operator
 
+Fluent Bit Operator defines five custom resources using CustomResourceDefinition (CRD):
 - **`FluentBit`**: Defines the Fluent Bit DaemonSet and its configs. A custom Fluent Bit image `kubesphere/fluent-bit` is required to work with FluentBit Operator for dynamic configuration reloading.
-- **`FluentBitConfig`**: Select input/filter/output plugins and generates the final config into a Secret.
-- **`Input`**: Defines input config sections.
-- **`Parser`**: Defines parser config sections.
-- **`Filter`**: Defines filter config sections.
-- **`Output`**: Defines output config sections.
+- **`ClusterFluentBitConfig`**: Select cluster-level input/filter/output plugins and generates the final config into a Secret.
+- **`ClusterInput`**: Defines cluster-level input config sections.
+- **`clusterParser`**: Defines cluster-level parser config sections.
+- **`ClusterFilter`**: Defines cluster-level filter config sections.
+- **`ClusterOutput`**: Defines cluster-level output config sections.
 
 Each **`Input`**, **`Parser`**, **`Filter`**, **`Output`** represents a Fluent Bit config section, which are selected by **`FluentBitConfig`** via label selectors. The operator watches those objects, constructs the final config, and finally creates a Secret to store the config. This secret will be mounted into the Fluent Bit DaemonSet. The entire workflow looks like below:
```

Comment:
Should add Cluster prefix to Input, parser, Filter, Output and FluentBitConfig here